src/main/java/org/sonar/plugins/csharp/CSharpSensor.java

```diff
@@ -219,8 +219,9 @@ private static String escapeXml(String str) {
   }
 
   private void importResults(SensorContext context) {
-    parseProtobuf(context, ProtobufImporters.issuesImporter(CSharpSonarRulesDefinition.REPOSITORY_KEY), ISSUES_OUTPUT_PROTOBUF_NAME);
+    // Note: the nosonar rules must be imported before issues, otherwise the affected issues won't get excluded!
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Is //NOSONAR used to exclude false-positive or to hide real quality flaw ? [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ANoSonar)